src/components/product/ProductList.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <section id="products">
+    <div class="container">
+      <h2 class="content-center">Featured</h2>
+      <div class="products-list">
+        <product-item
+          v-for="item in productList"
+          :key="item.id"
+          :item="item"
+          @edit="sendProductDataEditToModal"
+        ></product-item>
+      </div>
+    </div>
+    <modal-add-edit
+      v-if="toggleEditModal"
```

Comment:
Consider naming the variable something like `isEditModalVisible`.  
Toggle part implies that its some sort of an action... 🤔